qa/rpc-tests/bip135-threshold.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3
```

Comment:
rename without "-" and use logging in this file too, thx!

Reply:
ok, np